src/components/Form/Form.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <form class="">
+    <select class="" name="">
+      <option>Choose a State</option>
+      <option
+        v-for="(state, index) in states"
+        :key="index"
+        :value="state.state"
```

Comment:
This seems like a great solution at this point - just wondering if there is something more specific we should look to later as an id/key here?